examples/index.js

```diff
@@ -4,15 +4,15 @@ const FA = require('react-fontawesome')
 
 // Spinning
 ReactDOM.render(
-  <FA name='twitter' spin />,
+  <FA name="twitter" spin />,
   document.getElementById('spin')
 )
 
 // Stack
 ReactDOM.render(
-  <span className='fa-stack fa-lg'>
-    <FA name='square-o' stack='2x' />
-    <FA name='twitter' stack='1x' />
+  <span className="fa-stack fa-lg">
+    <FA name="square-o" stack="2x" />
```

Comment:
I changed those since that's the style enforced by your current linter configuration. Let me know how you want to fix it.